crates/bls12381/Cargo.toml

```diff
@@ -19,8 +19,8 @@ num-bigint = { workspace = true, features = ["rand"] }
 num-integer = { workspace = true }
 num-traits = { workspace = true}
 rand = { workspace = true}
-bls12_381 = { git = "https://github.com/wrinqle/bls12_381" }
+bls12_381 = { git = "https://github.com/wrinqle/bls12_381", features = ["experimental"] }
```

Comment:
@wwared Please open an issue to sort out hosting this on a Lurk fork.

Reply:
There's already #21 tracking this fork issue